### PR TITLE
automake: 1.16.3 -> 1.16.5

### PR DESCRIPTION
**Pull request by trofi**

### CODE REVIEW

File: pkgs/development/tools/misc/automake/automake-1.16.x.nix [hunk 1]
```diff
@@ -3,13 +3,24 @@
...
+    sha256 = "0v8kbw4fan7apbjwziylyjl5z708w452r5q6kpafcmssi04wryl0";
   };
 
+  patches = [
+    # Upstream patch to fix default PYTHON_PREFIX location.
+    # Will be included as a part of upcoming 1.16.5 release.
+    # Example failing package without the patch is 'rpm'.
+    # Not a 'fetchpatch' to avoid recursive bootstrap depends.
+    (fetchurl {
+      url = "https://git.savannah.gnu.org/cgit/automake.git/patch/m4/python.m4?id=563878435765c439182a4fe4dfb3158c31b2dad9";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/misc/automake/automake-1.16.x.nix:19]: I think we want to set name here.

**Comment by SuperSandro2000** [comment]
@ofborg build rpm

File: pkgs/development/tools/misc/automake/automake-1.16.x.nix (hunk 1 above)
Comment (trofi) [reply pkgs/development/tools/misc/automake/automake-1.16.x.nix:20]: Added `name = "python-prefix.patch";`.

File: pkgs/development/tools/misc/automake/automake-1.16.x.nix [hunk 2]
```diff
@@ -3,13 +3,25 @@
 stdenv.mkDerivation rec {
   # When updating, beware of https://github.com/NixOS/nixpkgs/pull/131928#issuecomment-896614165
```
[the remark this answers is not in the document]
Comment (SuperSandro2000) [reply pkgs/development/tools/misc/automake/automake-1.16.x.nix:4]: You didn't push the change.

[the remark this answers is not in the document]
Comment (trofi) [reply pkgs/development/tools/misc/automake/automake-1.16.x.nix]: Pushed now.

**Comment by SuperSandro2000** [comment]
mailutils and rpm build now so at least that regression is fixed. Lets give this a shot!

**Pull request merged**